graphql-dgs/src/test/kotlin/com/netflix/graphql/dgs/DgsSchemaProviderTest.kt

```diff
@@ -477,29 +481,66 @@ internal class DgsSchemaProviderTest {
                 val coordinates = FieldCoordinates.coordinates("Query", "myField")
                 return codeRegistryBuilder.dataFetcher(coordinates, df)
             }
+
+            // Result should be processed by DataFetcherResultProcessors
+            @DgsCodeRegistry
```

Comment:
Would you be able to also update our docs to use the DgsCodeRegistryBuilder? Much appreciated.

Reply:
Ack

Reply:
Actually, where are the docs :smile: ?